src/controllers/api/inventoryController.ts

```diff
@@ -1,11 +1,28 @@
-import inventory from "@/static/fixed_responses/inventory.json";
+/* eslint-disable @typescript-eslint/no-misused-promises */
+import { toInventoryResponse } from "@/src/helpers/inventoryHelpers";
+import { Inventory } from "@/src/models/inventoryModel";
 import { Request, RequestHandler, Response } from "express";
 
-const inventoryController: RequestHandler = (request: Request, response: Response) => {
-    console.log(request.query);
+const inventoryController: RequestHandler = async (request: Request, response: Response) => {
     const accountId = request.query.accountId;
+
+    if (!accountId) {
+        response.status(400).json({ error: "accountId was not provided" });
+        return;
+    }
     console.log(accountId);
-    response.json(inventory);
+
+    const inventory = await Inventory.findOne({ accountOwnerId: accountId }); // has the accountOwnerId field to find a corresponding inventory
```

Comment:
i dont think the comment is necessary
```suggestion
    const inventory = await Inventory.findOne({ accountOwnerId: accountId });
```

Reply:
it is important, because it is not present in the original response.

Reply:
this comment doesn't explain nothing, it says its using accountOwnerId to find an inventory which is obvious.

Reply:
it explains a lot.

I added the field accountOwnerId and everyone who knows the inventory structure would know the field is not originally there.

The purpose of the field is explained: it exists to find an account's corresponding inventory.

Reply:
should that belong then inside inventoryModel.ts?